config/settings.yml

```diff
@@ -663,4 +663,8 @@ paging_schedule:
 hold_instead_of_recall:
   - 'In process'
   - 'On order'
-  - 'In transit'
\ No newline at end of file
+  - 'In transit'
+
+unallowed_visitor_pickups:
+  - 'BUS-IDESK'
+  - 'LAW'
```

Comment:
I think we probably want to allow-list the ok pickup locations? I think there's only a couple pickup locations that can also handle visitor registration, right?